docs/admin/identity-access-management.md

```diff
@@ -0,0 +1,23 @@
+# Identity Access Management
+
+## Role concept
+
+The endpoints of the SSI Dim Middle Layer can only be used with an authenticated user. To be able to authorize the user, he needs to have the Audience `DIM-Middle-Layer` which is configured in the appsetting under `JWTBEAREROPTIONS__TOKENVALIDATIONPARAMETERS__VALIDAUDIENCE.
```

Comment:
the audience is configurable https://github.com/SAP/ssi-dim-middle-layer/blob/main/charts/dim/values.yaml#L197, here it seem like it's static, please change that and mention that a client needs to be created to authenticate against.